adhocracy4/comments_async/static/comments_with_categories/comment.jsx

```diff
@@ -102,6 +103,10 @@ export default class Comment extends React.Component {
     return this.props.content_type !== this.context.comments_contenttype
   }
 
+  commentCategoryChoices () {
+    return this.props.content_type !== this.context.comments_contenttype
+  }
```

Comment:
Why do we need this again? Could we just use displayCategories (which is the same) here or will this return the actual categories?

Reply:
This might be one of the things I need to tidy, there were alot of failed attempts, this might be one of them, I will have a look over it all

Reply:
You already fixed it in your next commit. :)